verifier/helpers.go

```diff
@@ -34,6 +34,26 @@ var VerificationPluginHeaders = []string{
 
 var errExtendedAttributeNotExist = errors.New("extended attribute not exist")
 
+// ValidateCerts ensures certs from input are CA certificates or self-signed.
+// It's designed to be used when user implements their own
+// truststore.X509TrustStore.
+func ValidateCerts(certs []*x509.Certificate) error {
```

Comment:
Should the following implementation also use this function to avoid duplicated code?
https://github.com/notaryproject/notation-go/blob/5cf34ace712920f8455857a5d7397ba49f4c00b1/verifier/truststore/truststore.go#L107

Reply:
Reformed the code. Moved it into the truststore package as it's a function for validating certs in a trust store.